src/Epic/contrib.index.tsx

```diff
@@ -0,0 +1,217 @@
+        }
+        ::-moz-selection {
+            background: ${palette.brandAlt[400]};
+        }
+    }
+    b {
+        font-weight: bold;
+    }
+`;
+
+const headingStyles = css`
+    ${headline.xxsmall({ fontWeight: 'bold' })}
+    margin-top: 0;
+    margin-bottom: ${space[3]}px;
+`;
+
+// Custom styles for <a> tags in the Epic content
+const linkStyles = css`
+    a {
+        color: ${palette.news[400]};
+        text-decoration: none;
+    }
+    a:hover {
+        text-decoration: underline;
+    }
+`;
+
+const bodyStyles = css`
+    margin: 0 auto ${space[2]}px;
+    ${body.medium()};
+    ${linkStyles}
+`;
+
+const highlightWrapperStyles = css`
+    ${body.medium({ fontWeight: 'bold' })}
+    ${linkStyles}
+`;
+
+const highlightStyles = css`
+    padding: 2px;
+    background-color: ${palette.brandAlt[400]};
+`;
+
+type HighlightedProps = {
+    highlightedText: string;
+};
+
+type BodyProps = {
+    paragraphs: string[];
+    highlightedText?: string;
+    countryCode?: string;
+};
+
+interface EpicHeaderProps {
+    text: string;
+}
+
+const EpicHeader: React.FC<EpicHeaderProps> = ({ text }: EpicHeaderProps) => {
+    return <h2 css={headingStyles}>{text}</h2>;
+};
+
+const Highlighted: React.FC<HighlightedProps> = ({ highlightedText }: HighlightedProps) => {
+    return (
+        <strong css={highlightWrapperStyles}>
+            {' '}
+            <span css={highlightStyles}>{highlightedText}</span>
+        </strong>
+    );
+};
+
+interface EpicBodyParagraphProps {
+    paragraph: string;
+    highlighted: JSX.Element | null;
+}
+
+const EpicBodyParagraph: React.FC<EpicBodyParagraphProps> = ({
+    paragraph,
+    highlighted,
+}: EpicBodyParagraphProps) => {
+    return (
+        <p css={bodyStyles}>
+            {paragraph}
+            {highlighted ? highlighted : null}
+        </p>
+    );
+};
+
+const EpicBody: React.FC<BodyProps> = ({ paragraphs, highlightedText }: BodyProps) => {
+    return (
+        <>
+            {paragraphs.map((paragraph, idx) => {
+                const paragraphElement = (
+                    <EpicBodyParagraph
+                        key={idx}
+                        paragraph={paragraph}
+                        highlighted={
+                            highlightedText && idx === paragraphs.length - 1 ? (
+                                <Highlighted highlightedText={highlightedText} />
+                            ) : null
+                        }
+                    />
+                );
+
+                return paragraphElement;
+            })}
+        </>
+    );
+};
+
+export const ContributionsEpic: React.FC<EpicProps> = ({
+    brazeMessageProps,
+    countryCode,
+    submitComponentEvent,
+}: EpicProps) => {
+    const [hasBeenSeen, setNode] = useHasBeenSeen({ threshold: 0 }, true) as HasBeenSeen;
+
+    useEffect(() => {
+        if (hasBeenSeen) {
+            // sendEpicViewEvent(tracking.referrerUrl, stage);
+        }
+    }, [hasBeenSeen]);
```

Comment:
I think we can probably remove this `useEffect`, and also supporting stuff like `hasBeenSeen` from here (I think this is all related to reporting epic views back to a system that the contributions team use).